.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "\U0001F41B Bug report"
+---
+
+This library is considered "complete", please view bugs you might identify rather as features, or maybe as wrinkles of worn code.
```

Comment:
The scare quotes around “complete” are confusing. Maybe "feature complete", but without the scare quotes around it? Scare quotes throw meaning into question, without clearly indicating what the question is, or what the answer is, either.

> please view bugs you might identify rather as features, or maybe as wrinkles of worn code.

This is completely unclear. As in I actually truly don't understand what it says, no matter how closely I read it and how I hard I try to read y'all minds. The library is so mature it couldn't possibly have any bugs so I shouldn't even bother filing one? Or what?

I suggest taking a step back, thinking about what you're really trying to communicate here, and then come right out and say it, instead of sort of hinting at it.

Reply:
The text the PR adds to the "Maintained by" section of the README is much clearer than this.

Reply:
Thank you, I guess I tried to say "please consider bugs as features" in a funny way. I dropped that part now and kept to the wording from the readme.